backend/api/handlers/application/GetTutorApplicationHandler.go

```diff
@@ -0,0 +1,69 @@
+package application
+
+import (
+	"LiteracyLink.com/backend/api/model"
+	"LiteracyLink.com/backend/auth"
+	"LiteracyLink.com/backend/db"
+	"errors"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"golang.org/x/crypto/bcrypt"
+	"gorm.io/gorm"
+	"net/http"
+)
+
+// No idea if this is this will work tbh, but it looks like it could haha
+
+func GetTutorApplicationHandler(c *gin.Context) {
+	var request model.AppRequest
+	err := c.BindJSON(request)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"status":  "error",
+			"message": fmt.Sprintf("Error while binding tutor application request: %g", err),
+		})
+	}
+
+	dbc := c.MustGet("db").(*gorm.DB)
+	app, err := db.GetApp(request, dbc)
+	if err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"status":  "failed",
+				"message": fmt.Sprintf("no application exists"),
+			})
+			return
+		} else {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"status":  "failed",
+				"message": fmt.Sprintf("ERROR: %g", err),
+			})
+			return
+		}
+	}
+
+	err = ((string(app.ParentId) != request.Parent) || (string(app.ChildId) != request.Child) || (string(app.DesiredSemesterId) != request.Semester))
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"status":  "failed",
+			"message": fmt.Sprintf("wrong indentifying info"),
+		})
+	}
+
+	jwt, err := auth.GenerateJWT(app.AppForTutId)
```

Comment:
We don't need to generate JWT tokens here. The JWT token is really only for user authorization and the only place we should generate then is in the login to hand it to a client. JWT token will be sent by the client and that is how the service will now that they are an actual user